### PR TITLE
Add detailed install docs

### LINKED ISSUE #28: Add detailed install instructions
We should add something similar to the engine install instructions: https://github.com/src-d/engine/#2-install-sourced-engine

### PR DESCRIPTION
**Pull request by dpordomingo**
fix #28 

based on https://github.com/src-d/engine/#2-install-sourced-engine, as suggested on issue #28 desc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,65 @@
...
+$ tar -xvf path/to/sourced-ce_REPLACE-VERSION_REPLACE-OS_amd64.tar.gz
+$ sudo mv path/to/sourced-ce_REPLACE-OS_amd64/sourced /usr/local/bin/
+```
+
+#### on Windows
+
+*Please note that from now on we assume that the commands are executed in `powershell` and not in `cmd`.*
+
+Create a directory for `sourced.exe` and add it to your `$PATH`;
+_e.g. you need to run these commands in a powershell as administrator._
```
Comment (carlosms) [inline README.md]: I don't think `e.g.` fits well here. What about:

Create a directory for `sourced.exe` and add it to your `$PATH`, running these commands in a powershell as administrator.

Comment (dpordomingo) [reply README.md]: totally!
(pushed)

**Review by carlosms — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -2,17 +2,64 @@
...
+### Install Docker
+
+_Please note that Docker Toolbox is not supported neither for Windows nor for macOS. In case that you're running Docker Toolbox, please consider updating to newer Docker Desktop for Mac or Docker Desktop for Windows._
+
+Follow the instructions based on your OS:
+
+- [Docker for Ubuntu Linux](https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce-1)
+- [Docker for Arch Linux](https://wiki.archlinux.org/index.php/Docker#Installation)
+- [Docker for macOS](https://store.docker.com/editions/community/docker-ce-desktop-mac)
+- [Docker Desktop for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows). Make sure to read the [system requirements for Docker in windows](https://docs.docker.com/docker-for-windows/install/).
```
Comment (smacker) [inline README.md]: `Docker in windows` -> `Docker on Windows`

Comment (dpordomingo) [reply README.md]: thanks! fixed and push-forced :D

**Pull request merged**